lsm-tree.cabal

```diff
@@ -337,7 +341,7 @@ library xxhash
   other-modules:   FFI
   hs-source-dirs:  xxhash/src
   build-depends:
-    , base
+    , base        <5
```

Comment:
Do we need the constraint in base everywhere? I'm guessing one on lsm-tree and one on blockio-api suffices?

Reply:
Unfortunately, this is what `cabal check` wants

Reply:
Boo